src/app/ui/NewTaskForm.tsx

```diff
@@ -211,12 +217,21 @@ export const NewTaskForm = ({
               }}
               selectorType={SelectorType.ASSIGNEE_SELECTOR}
               buttonContent={
-                <Typography variant="bodySm" lineHeight="20px" sx={{ color: (theme) => theme.color.gray[600] }}>
-                  {truncateText(
-                    (assigneeValue as IAssigneeCombined)?.name ||
-                      `${(assigneeValue as IAssigneeCombined)?.givenName ?? ''} ${(assigneeValue as IAssigneeCombined)?.familyName ?? ''}`.trim(),
-                    TruncateMaxNumber.SELECTOR,
-                  )}
+                <Typography
+                  variant="bodySm"
+                  lineHeight="20px"
+                  sx={{
+                    color: (theme) => theme.color.gray[600],
+                    textOverflow: 'ellipsis',
+                    whiteSpace: 'nowrap',
+                    overflow: 'hidden',
+                    maxWidth: { xs: '60px', sm: '100px' },
+                  }}
+                >
+                  {assigneeValue
+                    ? (assigneeValue as IAssigneeCombined)?.name ||
+                      `${(assigneeValue as IAssigneeCombined)?.givenName ?? ''} ${(assigneeValue as IAssigneeCombined)?.familyName ?? ''}`.trim()
+                    : 'Assignee'}
```

Comment:
Font size of status selector, assignee selector, and due date are 12px in Figma. Here we have given 13px. Please kindly do the needful. 

In Figma - 

![Screenshot from 2024-07-08 10-53-51](https://github.com/copilot-platforms/tasks-app/assets/38468429/e830e04e-1bc1-4c79-8d2f-c6b418a1dd94)


In this PR - 

![Screenshot from 2024-07-08 10-55-16](https://github.com/copilot-platforms/tasks-app/assets/38468429/ce015aa4-cbf7-45e4-bc9f-46611d7da3ad)